cypress/support/hub-commands.ts

```diff
@@ -230,30 +229,11 @@ Cypress.Commands.add('cleanupCollections', (namespace: string, repo: string) =>
 });
 
 Cypress.Commands.add('createNamespace', (namespaceName: string) => {
-  cy.requestPost(hubAPI`/_ui/v1/namespaces/`, {
-    name: namespaceName,
-    groups: [],
-  });
+  cy.galaxykit('namespace create', namespaceName);
 });
 
-Cypress.Commands.add('getNamespace', (namespaceName: string) => {
-  cy.requestGet<HubItemsResponse<HubNamespace>>(
-    hubAPI`/_ui/v1/namespaces/?name=${namespaceName}`
-  ).then((itemsResponse) => {
-    if (itemsResponse.data.length === 0) {
-      cy.createNamespace(namespaceName);
-    } else {
-      cy.log('Namespace Exists');
-    }
-  });
-});
-
-// Cypress.Commands.add('deleteNamespace', (namespaceName: string) => {
-//   cy.galaxykit('namespace delete', namespaceName);
-// });
-
 Cypress.Commands.add('deleteNamespace', (namespaceName: string) => {
-  cy.requestDelete(hubAPI`/_ui/v1/namespaces/${namespaceName}/`);
+  cy.galaxykit('-i namespace delete', namespaceName);
```

Comment:
Do we need the `-i`?

Reply:
I'm not sure we can always guarantee that a namespace can be deleted:
* rejected collections will break it
* repositories with multiple retain versions will break it
* orphaned collections will break it

So I think we'll have to rely on an async cleanup tasks to make sure namespaces are gone.